sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
@toolforger : if the function can't easily be doctested, then the docs shouldn't be tested. Doctests are for ensuring the docs remain valid, not for testing actual functionality. Skipping hard to test docs is perfectly fine. Either way, @moorepants  is right here - this function is non-public and doesn't need any deprecation warning.